.env.rc0

```diff
@@ -0,0 +1,150 @@
+BLOCKSCOUT_WEB_REPLICAS=3
+# Increment this value everytime you redeploy blockscout. Or else the deployment will fail due to the
+# existing database.
+BLOCKSCOUT_DB_SUFFIX="1"
+BLOCKSCOUT_SUBNETWORK_NAME="RC0"
+
+CELOSTATS_SERVER_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celostats-server"
+CELOSTATS_SERVER_DOCKER_IMAGE_TAG="2d970c69c109ab4089bddfe50c8f75d872e6e3a3"
+CELOSTATS_FRONTEND_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celostats-frontend"
+CELOSTATS_FRONTEND_DOCKER_IMAGE_TAG="db16a1a4d9882cfed93ffa9f61cfa9f80cadfe18"
+CELOSTATS_TRUSTED_ADDRESSES=""
+CELOSTATS_BANNED_ADDRESSES=""
+CELOSTATS_RESERVED_ADDRESSES=""
+
+GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_NODE_DOCKER_IMAGE_TAG="e0be402d3b146682cfa1827b97b1833e7dc913e9"
+
+GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth-all"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_BOOTNODE_DOCKER_IMAGE_TAG="e0be402d3b146682cfa1827b97b1833e7dc913e9"
+
+CELOTOOL_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
+CELOTOOL_DOCKER_IMAGE_TAG="celotool-9548656d0411d8c3b3d525727e621442e21b055a"
+
+CELOCLI_STANDALONE_IMAGE_REPOSITORY="gcr.io/celo-testnet/celocli-standalone"
+CELOCLI_STANDALONE_IMAGE_TAG="0.0.30-beta2"
+
+# Schedule for an oracle deployed via celotool, expressed in crontab syntax
+# This schedule is "every 5th minute"
+ORACLE_CRON_SCHEDULE="*/5 * * * *"
+
+ORACLE_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/oracle"
+ORACLE_DOCKER_IMAGE_TAG="releasecandidate0"
+
+TRANSACTION_METRICS_EXPORTER_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
+TRANSACTION_METRICS_EXPORTER_DOCKER_IMAGE_TAG="transaction-metrics-exporter-9548656d0411d8c3b3d525727e621442e21b055a"
+
+ATTESTATION_SERVICE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/celo-monorepo"
+ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="attestation-service-9548656d0411d8c3b3d525727e621442e21b055a"
+
+GETH_EXPORTER_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet-production/geth-exporter"
+GETH_EXPORTER_DOCKER_IMAGE_TAG="ed7d21bd50592709173368cd697ef73c1774a261"
+
+# Genesis Vars
+NETWORK_ID=20200312
+CONSENSUS_TYPE="istanbul"
+BLOCK_TIME=5
+# Minimum epoch length is 24 hours
+EPOCH=17280
+LOOKBACK=12
+ISTANBUL_REQUEST_TIMEOUT_MS=10000
+
+# the number of load test clients that will be given funds in the genesis & migrations
+LOAD_TEST_CLIENTS=0
+# every 7.2 seconds, so that 500 transactions are sent by a client every hour
+# to simulate 10,000 total transactions per hour
+LOAD_TEST_TX_DELAY_MS=7200
+
+# the amount in cUSD wei to give faucet, load test, and attestation bot accounts
+FAUCET_CUSD_WEI=60000000000000000000000
+
+VALIDATORS=70
+PROXIED_VALIDATORS=35
+ELECTION_MIN_VALIDATORS=50
+TX_NODES=5
+# STATIC_IPS_FOR_GETH_NODES=false
```

Comment:
If we make this not `VM_BASED`, this and the `GETH_NODES_SSD_DISKS` probably needs to be uncommented